app/components/Session/SavedSessions.js

```diff
@@ -1,56 +1,88 @@
-        this.props.deleteSavedSession(index);
+        this.props.deleteSavedSession(uuid);
       }
     };
   }
 
-
+  sessionFromUUID (uuid) {
+    const {savedSessions} = this.props;
+    for (let session of savedSessions) {
+      if (session.uuid === uuid) {
+        return session;
+      }
+    }
+    throw new Error(`Couldn't find session with uuid ${uuid}`);
+  }
 
   render () {
     const {savedSessions, setCaps, capsUUID, switchTabs} = this.props;
 
-    return (<Row>
+    const columns = [{
+      title: 'Capability Set',
+      dataIndex: 'name',
+      key: 'name'
+    }, {
+      title: 'Created',
+      dataIndex: 'date',
+      key: 'date'
+    }, {
+      title: 'Actions',
+      key: 'action',
+      render: (text, record) => {
+        let session = this.sessionFromUUID(record.key);
+        return (
+          <div>
+            <Button icon='edit'
+             onClick={() => {setCaps(session.caps, session.uuid); switchTabs('new');}}
+             className={SessionCSS['edit-session']}
+            />
+            <Button icon='delete' onClick={this.handleDelete(session.uuid)}/>
+          </div>
+        );
+      }
+    }];
+
+    let dataSource = [];
+    if (savedSessions) {
+      dataSource = savedSessions.map((session) => {
+        return {
+          key: session.uuid,
+          name: (session.name || '(Unnamed)'),
+          date: moment(session.date).format('YYYY-MM-DD')
+        };
+      });
+    }
+
+    let onRowClick = (record) => {
+      let session = this.sessionFromUUID(record.key);
+      setCaps(session.caps, session.uuid);
+    };
+
+    return (<Row gutter={20}>
       <Col span={12}>
-        <table className={SessionCSS['saved-sessions']}>
-          <tbody>
-            {savedSessions && [...savedSessions].reverse().map((session, index) => {
-              index = savedSessions.length - index - 1;
-              return <tr key={index} className={session.uuid === capsUUID ? SessionCSS['active-session'] : ''} onClick={() => setCaps(session.caps, session.uuid)}> 
-                <td>
-                  <p>
-                  {session.name || 'Untitled'}
-                  &nbsp;&nbsp;
-                  {moment(session.date).format('YYYY-MM-DD')}
-                  </p>
-                </td>
-                <td>
-                  {
-                    session.uuid === capsUUID &&
-                    <div className={SessionCSS['pull-right']}>
-                      <Button icon='edit' onClick={() => switchTabs('new')} className={SessionCSS['edit-session']}/>
-                      <Button icon='delete' onClick={this.handleDelete(index)}/>
-                    </div>
-                  }
-                </td>
-              </tr>;
-            })}
-          </tbody>
-        </table>
+        <Table className={SessionCSS['saved-sessions']}
+         pagination={false}
+         dataSource={dataSource}
+         columns={columns}
+         onRowClick={onRowClick}
```

Comment:
yes, this is a good idea. I overlooked this and couldn't find an easy way to do the highlighting.